tests/test_examples.py

```diff
@@ -510,6 +510,83 @@ def test_NAE_QSC_solve():
         )
 
 
+@pytest.mark.regression
+@pytest.mark.solve
+@pytest.mark.slow
+def test_NAE_QSC_2nd_order_solve():
```

Comment:
can you add a test for non-stellarator symmetry?

Reply:
I will add a warning/error